### PR TITLE
project setup done

### PR DESCRIPTION
**Pull request by majidali129**
project setup done

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1,13 @@
-<p>This is the first line of code</p>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- <link rel="icon" type="image/svg+xml" href="/vite.svg" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FILMPIRE</title>
```
Comment (majidali129) [inline index.html:7]: make sure to change and write a descriptive title

Comment (mubarrah01001) [inline index.html]: make sure to write ' The World Of Adventures ' instead of 'FILMPIRE' as a title

**Review by mubarrah01001 — CHANGES_REQUESTED** [review]
please change the code as my comment in review

**Review by mubarrah01001 — APPROVED** [review]
well done Majid Ali

**Pull request merged**